### DISCUSSION TITLE
Is this enclosure large enough for the Pi stack & other low voltage needs?

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#871] [Q&A]
I am in the process of building a Nixie to replace an ET4.  I am following in the footsteps & guidance of this thread: https://github.com/tagyoureit/nodejs-poolController/discussions/801.  I anticipate needing an auxiliary box for the Pi, terminal blocks, and all that will house all of the low voltage stuff.  The high voltage transformers, relays, breakers, and salt generator stuff will remain in the current ET box.

Will this box be sufficient for the Pi & its  friends, with the possible  addition of a Meanwell 24 volt DC transformer that can be DIN rail mounted?  (Hoping I find a bridge rectifier that does the job for me, but I'd like to get a box that's big enough for a transformer, just in case.)  I know a transformer will need a 120 volt cable back to the existing box.  I will run a separate conduit, if the transformer is needed.

https://altelix.com/altelix-10x8x6-vented-industrial-din-rail-fiberglass-enclosure/

Thanks for your help & thoughts!

**Comment by tagyoureit** [comment]
10x8 is probably the smallest I would go... Check out MyAZPool's setup - https://github.com/tagyoureit/nodejs-poolController/discussions/515.  He has a lot going on but it's a 24x20x9.  Somewhere, @rstrouse has a picture of his box, too.  If you want to start adding on any other components besides the basics, it will be too small.

**Reply by FlaMike** [reply]
OK, I appreciate the feedback.  Go big or go  home, right :-)

I may give this one a shot, then.  Deep enough for transformers & 17x14x6 so much more elbow room.

https://altelix.com/altelix-17x14x6-vented-din-rail-polycarbonate-abs-weatherproof-nema-enclosure/

Once I get the basics going, I hope to dive into a REM Chem controller for the acid feeder &  salt cell.  I'll also add a pressure sensor for the sand filter.  Not much else to add to my pool beyond that.   It's a 1 body with a gas heater & lights.  Pretty basic overall, but a little more room in the box certainly won't hurt.
